src/test/shell/data/i18n/en.json

```diff
@@ -272,7 +272,9 @@
     "LANGUAGE_DISABLED": "Disabled",
     "TEST_INSTANCE": "Mark as test NMaaS instance",
     "SEND_APP_INSTANCE_FAILURE_EMAILS": "Send emails when app instance fails to deploy",
-    "SEND_APP_INSTANCE_FAILURE_EMAILS_LIST": "Emails list"
+    "SEND_APP_INSTANCE_FAILURE_EMAILS_LIST": "Emails list",
+    "DOMAIN_SELECTION_ENABLED" : "Registration domain selection"
```

Comment:
@kbeyro please update English text to "Domain selection during registration" and other translations accordingly